pcsx2/gui/AppCoreThread.cpp

```diff
@@ -376,23 +389,26 @@ void AppCoreThread::ApplySettings( const Pcsx2Config& src )
 				gameMemCardFilter = game.getString("MemCardFilter");
 			}
 
-			if (EmuConfig.EnablePatches) {
-				if (int patches = InitPatches(gameCRC, game)) {
+			if (EmuConfig.EnablePatches)
+			{
+				int patches = InitPatches(gameCRC, game);
+				int fixes = loadGameSettings(fixup, game);
+
+				if (patches)
+				{
 					gamePatch.Printf(L" [%d Patches]", patches);
 					PatchesCon->WriteLn(Color_Green, "(GameDB) Patches Loaded: %d", patches);
 				}
-				if (int fixes = loadGameSettings(fixup, game)) {
+				if (fixes)
 					gameFixes.Printf(L" [%d Fixes]", fixes);
```

Comment:
The intent was to remove any [potential warnings](http://stackoverflow.com/questions/5476759/compiler-warning-suggest-parentheses-around-assignment-used-as-truth-value) which might occur when you use a variable initialization on conditional statements. The same method used for ``numberLoadedCheats`` and the other variables.

**Example 1:** 
`` if(a = 1) std::cout<<"Hey";``

**Results**: ``6:14: warning: suggest parentheses around assignment used as truth value [-Wparentheses]``

**Example 2:** 
`` if((a = 1)) std::cout<<"Hey";``

**Results**: ``No warnings``
 
Also note that you [can't follow this method](http://stackoverflow.com/questions/1516919/declaring-and-initializing-a-variable-in-a-conditional-or-control-statement-in-c) to get rid of the warning in case of the variable ``fixes`` and also keep the current scope of the variable. hence I had to increase the scope a bit.